.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Nginx image creator
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+
+  create-and-push-image:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+      attestations: write
+      id-token: write
+
+    steps:
+    - name: CHECKOUT
+      uses: actions/checkout@v3
+
+    - name: INSTALL Node
+      working-directory: ./node
+      run: npm install
+
+    - name: RUN SuperTest API Testing
+      working-directory: ./node
```

Comment:
Let's specify default `working-directory` for a whole job to DRY

Reply:
It is a Don't repeat yourself principal in programming - https://en.wikipedia.org/wiki/Don%27t_repeat_yourself